package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "obsidian-local-rest-api",
-  "version": "3.2.0",
```

Comment:
Could you remove these version number bumps?  I handle those as part of the release process and will increment the version number following [semver guidelines](https://semver.org/) depending on what kinds of changes end up in that release.

Also, by the way, since this is a new *feature* instead of a *bugfix* this would be a minor version bump instead of a patch (i.e. 3.2.0 -> 3.3.0 instead of 3.2.0 -> 3.2.1).